packages/components/vite.config.ts

```diff
@@ -19,4 +19,7 @@ export default defineConfig({
       generateScopedName: cssModuleClassNameGenerator,
     },
   },
+  build: {
+    emptyOutDir: false,
+  },
```

Comment:
Muss das tatsächlich? Ist eigenltich hilfreich, da so der `dist` nie deprecated Stuff beinhalten kann, was eine Diskrepanz zwischen Dev und Pipeline verhindert.

Evtl. die JSON in einem anderen Ordner speichern